components/home/Header/styled.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components";
+
+interface ContainerProps {}
+export const Container = styled.div<ContainerProps>`
+  display: flex;
```

Comment:
We can use Chakra for common components like flex boxes. Goal is to write as little code as possible

Reply:
Sure, got it.